lib/seahorn/BvOpSem2TrackingRawMemMgr.cc

```diff
@@ -274,8 +265,9 @@ TrackingRawMemManager::memsetMetaData(MetadataKind kind, PtrTy ptr,
 }
 Expr TrackingRawMemManager::getMetaData(MetadataKind kind, PtrTy ptr,
                                         MemValTy memIn, unsigned int byteSz) {
-  return m_metadata_map.at(kind)->loadIntFromMem(ptr, memIn.getMetadata(kind),
-                                                 byteSz, 0 /* TODO: fix */);
+  return m_metadata_map.at(kind)->loadIntFromMem(
```

Comment:
this should be aligned load

Reply:
please add comment in the code for that. Probably should expose this functionality eventually, but I'm ok with the current solution

Reply:
Done